epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -421,30 +436,51 @@ int main(int argc, char **argv)
 
     for (int ievt = 0; ievt < nevt; ++ievt) // Loop over all events in this iteration
     {
-      if (verbose)
-      {
-        // Display momenta
-        const int ipagM = ievt/neppM; // #eventpage in this iteration
-        const int ieppM = ievt%neppM; // #event in the current eventpage in this iteration
-        std::cout << "Momenta:" << std::endl;
+      auto getMomentum = [&](std::size_t evtNo, int particle, int component) {
```

Comment:
I would stick to the same naming scheme used throughout the rest of the code:
- particle -> ipar (index up to npar)
- component -> ip4 (index up to np4)
- page -> ipagM (page for Momenta)
- evtNo: I would use ievt, but then it may be better to move this outside the loop on ievt to avoid human confusion